web/src/main/java/org/openmrs/web/filter/initialization/InitializationFilter.java

```diff
@@ -1133,6 +1141,13 @@ private void importTestDataSet(InputStream in, String connectionUrl, String conn
 		}
 	}
 	
+	private boolean checkIfDatabaseIsSupported(String database) {
```

Comment:
This method name creates an impression that we support only one database. But from the code base, we support four databases. Do you get my point?

Reply:
Does **checkInSupportedDatabases** looks right ?

Reply:
What do you think of `isCurrentDatabase`?

Reply:
Seems fine to me. I'll make the changes.